uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
```

Comment:
Thanks, what I implied is the same device, yes, not the same sensor. We should wait for Mr. Shamaev's comments on this.